contracts/factory/packages/FactoryPackage.sol

```diff
@@ -77,6 +83,22 @@ contract FactoryPackage is FactoryStorage, IFactory, IFactoryInit, IFactoryEvent
         return vaults;
     }
 
+    function getVaultPackages() external view override returns (address[] memory) {
```

Comment:
Do we have scenarios where we need this function? If not I think we should store vaultPackages as 
```
mapping(address => bool) isPackage
```
It will make duplication validation much cheaper. in this case  `getVaultPackage` function needs to be removed and `isVaultPackage(address) external view returns (bool)` added

Reply:
I did it like this to make it easier for the subgraph to index and then the frontend show it in the new Vault Management screen

Reply:
Subgraph listens to events to get data. This function is not needed for the subgraph to store added vaults.
So let's replace vaultPackages with `mapping(address => bool) isPackage` as I mentioned above